src/Prelude.purs

```diff
@@ -1,108 +1,29 @@
 module Prelude
-  ( Unit(), unit
-  , ($), (#)
-  , flip
-  , const
-  , asTypeOf
+  ( module P
```

Comment:
Yep! You need to `import qualified _ as _` though, `import _ as _` doesn't work which perhaps should be added too.